tox.ini

```diff
@@ -1,11 +1,24 @@
 [tox]
 skipsdist=True
-envlist =
-    py27-{yamllint,ansible_syntax,generate_validation}
+envlist = yamllint,ansible-syntax
 skip_missing_interpreters=True
 
 [testenv]
-skip_install=True
+deps = -r{toxinidir}/requirements.txt
 
-commands =
-    yamllint: yamllint examples/
+[testenv:yamllint]
+whitelist_externals =
+    bash
+commands = bash -c "\
+        yamllint \
+                -c {toxinidir}/.yamllint \
+                {toxinidir}/roles"
```

Comment:
I think we should check `tasks/` dir and not roles, no?

Reply:
Ah, sorry, it's infra role :)

Reply:
But still maybe we can check whole root...

Reply:
Also what's the reason of using `bash -c' ?

Reply:
yamllint will not check the improrted roles from there if I put tasks there, doens't care about import_role.

I can put both.

Reply:
I think the whole root, could be better?

Reply:
Sure, I 'll need to exclude some files like .travis.yml only.